Backend/api/views/transaction_view.py

```diff
@@ -171,6 +171,36 @@ def get(self, request):
             "transactions" : transaction_list
         })
 
+class GetAllRecievedTransactionsView(APIView):
+    permission_classes = (IsAuthenticated,)
+
+    def get(self, request):
+        user_id = request.user.id
+        addresses = Addresses.objects.filter(user_id=user_id).all()
+        
+        transaction_list =[]
+        for address in addresses:
+            transaction_request = Session().get(
+            url=f"https://blockstream.info/testnet/api/address/{address.address_generated}/utxo")
+            print(transaction_request.json())
+            transactions = [
```

Comment:
Why not just loop through the address table,any that has is_broadcasted=True should be returned, instead of getting the information through blockstream?

Reply:
Ohh, no. we can not do that cos it is a received transaction
What is currently in our DB is what we created ourselves i.e created transactions

Reply:
Oh, received.  I was thinking of broadcasted transaction